image/cli/masfvt/fvt-predict.yml

```diff
@@ -33,6 +34,7 @@
           - "fvt_artifactory_username .......... {{ fvt_artifactory_username }}"
           - "fvt_artifactory_token ............. {{ fvt_artifactory_token }}"
           - "fvt_digest_ctf .................... {{ fvt_digest_ctf }}"
+          - "fvt_digest_predict .................... {{ fvt_digest_predict }}"
```

Comment:
small one: remove some `...` to keep alignment pattern :)

Reply:
Fixed.